relay-event-schema/src/protocol/span.rs

```diff
@@ -67,6 +67,9 @@ pub struct Span {
     #[metastructure(pii = "true")]
     pub data: Annotated<Object<Value>>,
 
+    /// Tags generated by Relay. These tags are a superset of the tags set on span metrics.
+    pub sentry_tags: Annotated<Object<String>>,
```

Comment:
Note: We would normally have to upgrade the relay library in sentry when adding properties to prevent errors like [this](https://github.com/getsentry/sentry-react-native/issues/1174). But because `Span` still has `retain = true` on the `other` field, upgrading is not necessary in this case.